src/aiperf/common/config/user_config.py

```
@@ -0,0 +1,1131 @@
+        ),
+        CLIParameter(
+            name=("--no-gpu-telemetry",),
+            group=Groups.TELEMETRY,
+        ),
+    ] = False
+
+    _gpu_telemetry_mode: GPUTelemetryMode = GPUTelemetryMode.SUMMARY
+    _gpu_telemetry_collector_type: GPUTelemetryCollectorType = (
+        GPUTelemetryCollectorType.DCGM
+    )
+    _gpu_telemetry_urls: list[str] = []
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)